src/pages/about.js

```diff
@@ -1,66 +1,118 @@
+              than you ever thought possible.
+            </p>
 
-        <p className="text-left leading-loose mt-4">
-          Examples of behavior that contributes to creating a positive
-          environment include:
-        </p>
+            <p className="mb-5 font-bold">
+              The skills you practice, and the connections you make here, might
+              just change your career… and your life.
+            </p>
+            <p>
+              As the Nashville chapter of the freeCodeCamp.org global learning
+              community, we are laser-focused on supporting those just starting
+              their journey in web development. We learn together and challenge
+              each other to thrive in the tech industry. We offer weekly
+              educational opportunities and code challenges centered on the
+              freeCodeCamp open source curriculum, as well as a monthly meetup,
+              networking opportunities and more.
+            </p>
+          </div>
 
-        <ul className="text-left">
-          <li>Using welcoming and inclusive language</li>
-          <li>Being respectful of differing viewpoints and experiences</li>
-          <li>Gracefully accepting constructive criticism</li>
-          <li>Focusing on what is best for the community</li>
-          <li>Showing empathy towards other community members</li>
-        </ul>
+          <div className="flex-auto pt-20 hidden md:block">
+            <LearnTeachCoding
+              className="inline-block"
+              alt="An image of Five Key words (Learn, Create, Collaborate, Teach, And Grow) suround by curry brackets"
+            />
+          </div>
+        </div>
+      </section>
 
-        <p className="text-left leading-loose mt-4">
-          Examples of unacceptable behavior by participants include:
-        </p>
+      <section className="FCC-divider">
+        <h2>
+          <span>LEADERSHIP</span>
+        </h2>
+      </section>
 
-        <ul className="text-left">
-          <li>
-            The use of sexualized language or imagery and unwelcome sexual
-            attention or advances
-          </li>
-          <li>
-            Trolling, insulting/derogatory comments, and personal or political
-            attacks
-          </li>
-          <li>Public or private harassment</li>
-          <li>
-            Publishing others&apos; private information, such as a physical or
-            electronic address, without explicit permission
-          </li>
-          <li>
-            Other conduct which could reasonably be considered inappropriate in
-            a professional setting
-          </li>
-        </ul>
+      <section className="main-section bg-white">
+        <div className="main-section-content flex flex-col  md:justify-around ">
+          <div className="flex flex-col  justify-around items-center md:flex-row">
+            <Img
+              className="rounded-full w-64 m-3 md:w-1/4"
```

Comment:
Great job with the images! There's a few things to add:

1. `border-2 border-FCCgray-100` should add the border around the image.
2. It could use a greater right margin on the desktop version.
3. It could use a greater bottom margin on the mobile version.
4. Seth's picture is a bit off-center compared to the mock-up... but that's also how it appears in the image we were provided. Probably not a big deal, right?

Reply:
First, thanks for the outline catch. 👍 I forgot about that. I updated the margin to better match the mock-ups. For Seth's picture, I think it's fine.